src/components/table/Table.jsx

```diff
@@ -116,6 +117,15 @@ const Table = (props) => {
         {/*    count: tc.count,*/}
         {/*  })}*/}
         {/*</div>*/}
+        { exportUrl && (
+          <div className="mr-6">
+            <ExportXlsx
+              exportUrl={exportUrl}
+              params={tc.getExportUrlParams()}
```

Comment:
just use `getUrlParams`, don't duplicate methods
```suggestion
              params={tc.getUrlParams()}
```

Reply:
`getUrlParams()` contains default pagination. I need params for export without pagination.

> just use `getUrlParams`, don't duplicate methods

done